src/encoded/static/components/static-pages/placeholders/PageCarousel.js

```diff
@@ -0,0 +1,215 @@
+'use strict';
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import Carousel from 'nuka-carousel';
+import { console, layout } from '@hms-dbmi-bgm/shared-portal-components/es/components/util';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['console' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4476879)